qutebrowser/extensions/loader.py

```diff
@@ -118,6 +121,24 @@ def _walk_normal() -> typing.Iterator[ExtensionInfo]:
         yield ExtensionInfo(name=name)
 
 
+def walk_extensions() -> typing.Iterator[ExtensionInfo]:
+    """Walk external extensions."""
+    ext_dir = os.path.join(standarddir.data(), 'extensions')
+    if not os.path.exists(ext_dir):
+        os.mkdir(ext_dir)
+        return
+
+    for finder, name, ispkg in pkgutil.walk_packages(
+            path=[ext_dir],
+            prefix='extensions.',
+            onerror=_on_walk_error):
+        if ispkg:
+            continue
+        if name not in sys.modules:
+            _module = finder.find_module(name).load_module(name)
```

Comment:
Does this line do anything? If so, maybe document it, because it does not look like it does anything...

Reply:
Yes it imports the module and saves it into `sys.modules` as a side effect. I will add a comment and remove the left hand side.